### ISSUE TITLE
NPC: Twilight Eementalist - Might have been programmed as an elemental. (immune to bleeds)

### ISSUE THREAD

**Issue by Contrubius** [azerothcore/azerothcore-wotlk#11553]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [X] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59
- [ ] 60

### Current Behaviour
The NPC Twilight Elementalist in Blackfathom Deeps is immune to bleeds
![bleed immune](https://user-images.githubusercontent.com/100415626/165655529-9bcc95f0-c0e2-4858-9d3b-7387da4f9038.jpg)

![WoWScrnShot_042822_025056](https://user-images.githubusercontent.com/100415626/165655554-ee599284-4c22-46f6-b575-3753e200fb91.jpg)

### Expected Blizzlike Behaviour

NPC: Twilight Elementalist is a humanoid and should not be immune to bleeds.

### Source
[Asmongold Classic Stream using Rend and not being immune on Twilight Elementalists.](https://youtu.be/snSTzDUNO-s?t=2962)

https://classic.wowhead.com/npc=4814/twilight-elementalist
### Steps to reproduce the problem

1. Locate Twilight Elementalist NPC near the end of the Blackfathom Deeps dungeon.
2. Try to use Rend or any other bleed effect.
3. See the NPC is immune.

### Extra Notes
Originally reported: https://github.com/chromiecraft/chromiecraft/issues/3425

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/49191da0f8566a513a7313fa8215e67c7cb996ab

### Operating system

Windows 10.

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [mod-save-inventory](https://github.com/Nyeriah/mod-save-inventory)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Gultask** [comment]
https://youtu.be/-887Zu5hqJo?t=4646

I found another source to, this time about the Rake, and it also shows that they're not immune to stuns.
From what I've gathered this particular NPC is very peculiar, being immune to nearly everything, see: https://github.com/chromiecraft/chromiecraft/issues/1333
The only thing that stopped changing this for the Elementalist, however, was a lack of sources.

See https://github.com/azerothcore/azerothcore-wotlk/issues/5915 for the whole discussion.
